docs/docs/rest-api/public/api/v2/schema/AppDefinition.json

```diff
@@ -120,7 +120,7 @@
           },
           {
             "type": "string",
-            "enum": ["UNIQUE", "CLUSTER", "GROUP_BY", "LIKE", "UNLIKE", "MAX_PER"]
+            "enum": ["UNIQUE", "CLUSTER", "GROUP_BY", "LIKE", "UNLIKE", "MAX_PER", "IS"]
```

Comment:
Could you rebase or merge on master. I don't think this should be part of your diff.